### PR TITLE
Mayhem

### PR DESCRIPTION
**Pull request by jacobclemente**

### CODE REVIEW

File: fuzz.c [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#include "id3v2lib.h"
```
Comment (xansec) [inline fuzz.c:1]: I would recommend changing the name of this file and target to indicate what is actually being fuzzed.

**Review by xansec — CHANGES_REQUESTED** [review]
Please make the requested changes.

**Comment by jacobclemente** [comment]
I have made the appropriate changes, but have run into a bit of a snag with getting the action to succeed. I recently changed my Github username to omit a hyphen, and I could not use Mayhem do to a mismatched username. I was unable to change my username on the ForAllSecure website, so I resorted to deleting the old account. I am having trouble signing into my newly created account, linked to my Github.

**Comment by jacobclemente** [comment]
I have implemented the requested changes to my fork's Mayhem branch.

**Comment by xansec** [comment]
LGTM!

**Pull request merged**